integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7323,6 +7327,29 @@ msgstr "%(used)s von %(total)s Wörtern bereits verbraucht"
 msgid "Next reset date:"
 msgstr "Nächstes Rücksetzungsdatum:"
 
+#: cms/templates/translations/translations_management.html
+msgid "Number of words in the German contents"
+msgstr "Anzahl der Wörter in den deutschen Inhalten"
+
+#: cms/templates/translations/translations_management.html
+msgid ""
+"The number of words in the latest versions per status and per content type."
+msgstr ""
+"Die Anzahl der Wörter in den neuesten Versionen pro Status und pro "
+"Intahltstype."
+
+#: cms/templates/translations/translations_management.html
+msgid "Content type"
+msgstr "Inhaltstyp"
+
+#: cms/templates/translations/translations_management.html
+msgid "Total per Content type"
+msgstr "Summe pro Inhaltstyp"
```

Comment:
Besides the typo, I think it might be a tiny bit easier to grasp at a glance with this:
```suggestion
#: cms/templates/translations/translations_management.html
msgid ""
"The number of words in the latest versions per status and per content type."
msgstr ""
"Die Anzahl der Wörter in den neuesten Versionen pro Status und pro "
"Art des Inhalts."

#: cms/templates/translations/translations_management.html
msgid "Content type"
msgstr "Art des Inhalts"

#: cms/templates/translations/translations_management.html
msgid "Total per Content type"
msgstr "Summe pro Art des Inhalts"
```